proxy/package.json

```diff
@@ -12,20 +12,23 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@bentley/bentleyjs-core": "^2.9.9",
-    "@bentley/express-server": "^2.9.9",
-    "@bentley/itwin-client": "^2.9.9",
+    "@bentley/bentleyjs-core": "^2.19.1",
+    "@bentley/express-server": "^2.19.1",
+    "@bentley/itwin-client": "^2.19.1",
     "axios": "^0.21.0",
     "dotenv": "^8.2.0",
+    "dotenv-expand": "^5.1.0",
+    "dotenv-flow": "^3.2.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "openid-client": "^4.2.2",
-    "ts-node-dev": "^1.1.1"
+    "openid-client": "^4.2.2"
   },
   "devDependencies": {
+    "@types/dotenv-flow": "^3.1.0",
     "@types/express": "^4.17.9",
     "@types/jsonwebtoken": "^8.5.0",
     "rimraf": "^2.6.2",
+    "ts-node-dev": "^1.1.1",
```

Comment:
What's this one for? Does it do a tsc watch and run the server.

Reply:
yea, exactly. was previously listed as a direct dep, but should be a dev dep.